starlark/experimental/go.star

```diff
@@ -123,7 +133,7 @@ def ko(task_name, git_name, image_repo, path, tag="$(context.build.name)", ldfla
                 path
             ],
             env=env,
-            workingDir=git_checkout_dir(git_name),
+            workingDir=build_working_dir,
```

Comment:
Could you add a `join` function in `starlark/stable/path.star` and use it here?

```python
def join(path, *paths):
    """Join two or more path components, inserting '/' as needed.
    If any component is an absolute path, all previous path components
    will be discarded."""

    for p in paths:
        if p.startswith("/"):
            path = p
        elif p != "":
            path += ("" if path == "" or path.endswith("/") else "/") + p
    return path
```
Then use it here as `workingDir=join(git_checkout_dir(git_name), working_dir)`?
If so, you could default `working_dir` to `""`, and remove the `build_working_dir` above.